### ISSUE TITLE
CustomTheme broken?

### ISSUE THREAD

**Issue by aschuler84** [evilz/vscode-reveal#126]
Hi, 
I'm using vs-code version 1.31.0 together with current vscode-reveal 3.2.0  on MacOsx Mojave.

However there seems to be a problem with `customTheme` entries. When I use a `cusomTheme` in the preamble of my slides the theme is not loaded in preview or browser preview. The slides fall back to the default theme. 

The css-File for the custom theme I want to use is the same directory as my markdown slides. Filename is `custom.css`, preamble of my markdown document is 

```
---
customTheme : "custom"
transition: "fade"
highlightTheme: "obsidian"
---
```

I don't know if that has something to do with the the issue stated above, however it also seems that images cannot be found - as preview and browser preview cannot load and display images.

When switching back to version 2.0.0 (Preview) form vscode market-place everything works fine. Thanks for your help.

**Comment by evilz** [comment]
Thank so much for this one !!!!
I'm on it.  Waiting CI to release :)

**Comment by evilz** [comment]
Should be fixed now in v3.3.0. Can you confirm it to me.

Thank

**Comment by aschuler84** [comment]
Works again with version v3.3.0, thank you very much!
